rakiety.py

```diff
@@ -1,20 +1,28 @@
 import requests
 import csv
-class rakiety:
-    def __init__(daj, flight_number, mission_name, rocket_id, rocket_name, launch_date_utc, video_link):
-        daj.rocket_id = rocket_id
-        daj.rocket_name = rocket_name
-        daj.launch_date_utc = launch_date_utc
-        daj.video_link = video_link
-        daj.mission_name = mission_name
-        daj.flight_number = flight_number
-lista = []
-link = requests.get("https://api.spacexdata.com/v3/launches").json()
-for flight in link:
-    lista.append(rakiety(flight['rocket']['rocket_id'], flight['mission_name'], flight['rocket']['rocket_id'], flight['rocket']['rocket_name'], flight['launch_date_utc'], flight['links']['video_link']))
+class Rockets:
+    def __init__(self, flight_number, mission_name, rocket_id, rocket_name, launch_date_utc, video_link):
+        self.rocket_id = rocket_id
+        self.rocket_name = rocket_name
+        self.launch_date_utc = launch_date_utc
+        self.video_link = video_link
+        self.mission_name = mission_name
+        self.flight_number = flight_number
+list = []
+class ConnectionError(Exception):
+    pass
+def Get_SpaceX_data(url):
+    try:
+        response = requests.get(url)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        raise ConnectionError("Connection Error")
+    return response.json()
+for flight in Get_SpaceX_data("https://api.spacexdata.com/v3/launches"):
+    list.append(Rockets(flight['rocket']['rocket_id'], flight['mission_name'], flight['rocket']['rocket_id'], flight['rocket']['rocket_name'], flight['launch_date_utc'], flight['links']['video_link']))
 
 with open("rakiety.csv", "w") as f:

```

Comment:
```suggestion
with open("rakiety.csv", "w") as file:
```
staramy sie unikac uzywania zmiennych ktore nic nie mowia